lib/presentation/redux/states/my_space_state.dart

```diff
@@ -53,10 +53,14 @@ class MySpaceState extends LinShareState {
   }
 
   @override
-  LinShareState sendViewState({Either<Failure, Success> viewState, List<Document> documentList}) {
+  LinShareState sendViewState({Either<Failure, Success> viewState}) {
     return MySpaceState(viewState, documentList);
```

Comment:
hmm, this `documentList` is not totally immutable. Ok. let move on, we need to find the new way to work with collection

Reply:
Why is it not totally immutable ? The only way to update it, it's via `setDocuments` I think it's ok

Reply:
just a question. We can add or remove item to list?

Reply:
We can not add or remove item from a view state because we have `final List<Document> documentList`. the only way to do it is create a new state with new document list

Reply:
are you sure? IMO `documentList` is final but the item of it is not. But I am not sure. Please ignore it at this time. We will focus on it next sprint.

Reply:
documentList is final but it can be set, however we can't remove or add one item without changing all items.

Reply:
about this state, it is ok, but I concerned the way we work with list. That a notice only. No need to change anythings. Move on!!!!!